packages/ts/form/src/Binder.ts

```diff
@@ -10,7 +12,7 @@ import type { AbstractModel, ModelConstructor } from './Models.js';
  * @typeParam T - Type of the value that binds to a form
  * @typeParam M - Type of the model that describes the structure of the value
  */
-export class Binder<T, M extends AbstractModel<T>> extends BinderRoot<T, M> {
+export class Binder<M extends AbstractModel> extends BinderRoot<M> {
```

Comment:
This looks like a breaking change. Assuming that I have some code like this in my application:
```ts
private doSomethingWithBinder(binder: Binder<Person, PersonModel>) { ... }
```
It will not compile anymore.